### DISCUSSION TITLE
Streaming links cannot be copied anymore since last update

### DISCUSSION THREAD

**Discussion by Fire222222** [aclap-dev/video-downloadhelper#188] [Help and Q&A] [answered]
### Your question

Hi,

Since latest update it is not possible anymore to copy the streaming links after video is playing.

For me it is very important to use as the original streaming links are much faster (e. g. by downloading with VLC) than downloading streams in VDH. e. g. Downloading Streaming Link with VLC: 11 MByte/sec. and same with VDH: 4,5 MByte/sec.

Will this feature return?

### Addon & CoApp version and details

Video DownloadHelper 8.2.0.23 (stable) licensed
VdhCoApp 2.0.10
Windows 10 64-bit, Firefox 123.0

**Comment by paulrouget** [comment, marked as answer]
This has been fixed in version 8.2.1, which should come out soon (waiting for mozilla to push the update).

**Reply by Fire222222** [reply]
Thanks!
